certbot/display/util.py

```diff
@@ -126,9 +140,9 @@ def menu(self, message, choices, ok_label="OK", cancel_label="Cancel",
                 (str(i), choice) for i, choice in enumerate(choices, 1)
             ]
             # pylint: disable=star-args
-            code, index = self.dialog.menu(message, **menu_options)
+            code, index = _clean(self.dialog.menu(message, **menu_options))
 
-            if code == CANCEL:
+            if code == CANCEL or index == "":
```

Comment:
Question: Is this necessary after checking the exit code? Is it possible to have a OK or HELP exit code and an empty index string?